### DISCUSSION TITLE
[Projects Beta] Bug: Only first 12 labels shown in sidebar Labels list

### DISCUSSION THREAD

**Discussion by gabriel-v** [community/community#50837] [Projects and Issues]
### Select Topic Area

Bug

### Body

We have more than 40 labels, numbered and sorted by category.

When trying to add labels to a new issue, only the first 12 are shown in the list.

The list does not infinite scroll, or fetch more labels. This means one has to manually type in the prefix of the label, instead of just scrolling through.

Projects Classic had this feature working: the labels list would display all possible labels when scrolled through.

### Projects Beta: 12 labels max
![Screenshot from 2023-03-23 18-37-14](https://user-images.githubusercontent.com/7493327/227273076-63751550-d271-444c-9052-faa275ae73ff.png)

### Projects Classic: All Labels
![Screenshot from 2023-03-23 18-38-21](https://user-images.githubusercontent.com/7493327/227273457-7049b6f6-e6c9-4750-8ce3-278615f32b7d.png)

### Steps to reproduce:
- add more than 20 labels to project
- try to add label to new issue
- see only 12 labels out of 20
- scrolling does not load more labels

**Comment by danudesigns** [comment]
Also encountered this issue. 12 label limit. 
Glad to find this already open. Thanks @gabriel-v ! 

- Lazy loading labels on scroll would be nice.
